ROADMAP.md

```diff
@@ -25,29 +26,16 @@ For a more detailed view see https://github.com/orgs/libp2p/projects/4/views/1.
 
 <!-- markdown-toc end -->
 
-## TLS support
-
-| Category     | Status      | Target Completion | Tracking                                        | Dependencies                       | Dependents |
-|--------------|-------------|-------------------|-------------------------------------------------|------------------------------------|------------|
-| Connectivity | In progress | Q4/2022           | https://github.com/libp2p/rust-libp2p/pull/2945 | [QUIC](#experimental-quic-support) |            |
-
-
-This allows us to secure both TCP and QUIC connections using TLS. This is a requirement for QUIC
-support. Running TLS on top of TCP is a nice to have, since we already have noise.
-
-## Experimental QUIC support
+## QUIC support
 
 | Category     | Status      | Target Completion | Tracking                                          | Dependencies                                                        | Dependents |
 |--------------|-------------|-------------------|---------------------------------------------------|---------------------------------------------------------------------|------------|
 | Connectivity | In progress | Q4/2022           | https://github.com/libp2p/rust-libp2p/issues/2883 | [TLS](#tls-support), https://github.com/libp2p/test-plans/issues/53 |            |
```

Comment:
I can't add a suggestion since this has been committed, but "[TLS](#tls-support)" is no longer a valid section.

Reply:
Addressed in https://github.com/libp2p/rust-libp2p/pull/3091